impeller/BUILD.gn

```diff
@@ -9,10 +9,6 @@ config("impeller_public_config") {
 
   defines = []
 
-  if (impeller_supports_platform) {
-    defines += [ "IMPELLER_SUPPORTS_PLATFORM=1" ]
```

Comment:
Do we need to remove references to this from the code?

Reply:
Yeah. I built the unit-tests to verify. Going over the code to change these references.

Reply:
Done.